receiver/purefareceiver/config.go

```diff
@@ -32,8 +32,8 @@ type Config struct {
 	// Settings contains settings for the individual scrapers
 	Settings *Settings `mapstructure:"settings"`
 
-	// Arrays represents the list of arrays to query
-	Arrays []internal.ScraperConfig `mapstructure:"arrays"`
+	// Array represents the list of arrays to query
```

Comment:
This comment makes me think array should remain plurals, as its a list of arrays. if i read correctly, the endpoint is mapped to `/array`, but as a user, i wouldn't know which of these configuration options need to be plural vs singular if some singular and plural need lists as configurations.

Reply:
Excellent review @codeboten I think that was surgical. But the comment that Christopher just sent, resolves this ambiguity for the config.